src/palace/manager/integration/license/overdrive/script.py

```diff
@@ -174,3 +175,74 @@ def explain_advantage_collection(
             print(" ", book["title"])
             if i > 10:
                 break
+
+
+class ImportCollection(Script):
+    """A convenient script for manually kicking off an OverDrive collection import"""
+
+    @classmethod
+    def arg_parser(cls) -> argparse.ArgumentParser:
+        parser = argparse.ArgumentParser()
+        parser.add_argument(
+            "--collection-name",
+            type=str,
+            help="Collection Name",
+        ),
```

Comment:
Minor: Why the trailing comma?

```suggestion
        )
```